dubbo-common/src/main/java/org/apache/dubbo/common/utils/ConfigUtils.java

```diff
@@ -259,7 +259,7 @@ public static Properties loadProperties(String fileName, boolean allowMultiFile,
             if (list.size() > 1) {
                 String errMsg = String.format("only 1 %s file is expected, but %d dubbo.properties files found on class path: %s",
                         fileName, list.size(), list.toString());
-                logger.warn(errMsg);
+                throw new IllegalStateException(errMsg);

```

Comment:
To avoid the following sentence from being executed?

`properties.load(ClassUtils.getClassLoader().getResourceAsStream(fileName));`

https://github.com/apache/dubbo/pull/7978/files#diff-04e43b4a240d7dc90d1b515a05af5baca07e3e96e3d2fae4c7b14722d0b4e145R267

Reply:
yes, It looks like what you said.

I see that the code record from a long time ago threw an exception, and then it was deleted, but I don’t know why, I can’t open the attached connection here.

http://code.alibabatech.com/jira/browse/DUBBO-133

Reply:
I mean if `properties.load(ClassUtils.getClassLoader().getResourceAsStream(fileName));` can work well with multiple properties file, we'd better not not add this restriction.

Reply:
OK